west.yml

```diff
@@ -188,7 +188,7 @@ manifest:
       groups:
         - hal
     - name: hal_nordic
-      revision: 54bde38c6f6ffb3780b26ae728cf79426184384e
+      revision: pull/254/head
```

Comment:
Please change to https://github.com/zephyrproject-rtos/hal_nordic/pull/261 instead, to include newest fixes.
(https://github.com/zephyrproject-rtos/hal_nordic/pull/254 is closed now)

can be done by rebasing from: https://github.com/zephyrproject-rtos/zephyr/pull/81710